testSideCar is failing

`testSideCar` test is failing on our CI based on openshift. I am not really sure if it's an openshift issue or a test issue.

# Expected Behavior

Working tests

# Actual Behavior

```
=== RUN   TestSidecarTaskSupport/A_sidecar_that_runs_forever_is_terminated_when_Steps_complete
--- FAIL: TestSidecarTaskSupport (59.28s)
    init_test.go:113: Create namespace arendelle-g99jz to deploy to
    init_test.go:125: Verify SA "default" is created in namespace "arendelle-g99jz"
    --- FAIL: TestSidecarTaskSupport/A_sidecar_that_runs_forever_is_terminated_when_Steps_complete (58.74s)
        sidecar_test.go:84: Creating Task "sidecar-test-task-0"
        sidecar_test.go:89: Creating TaskRun "sidecar-test-task-run-0"
        sidecar_test.go:132: Sidecar container did not terminate as expected, Terminated state: &ContainerStateTerminated{ExitCode:137,Signal:0,Reason:Error,Message:,StartedAt:2019-08-30 08:49:51 +0200 CEST,FinishedAt:2019-08-30 08:50:30 +0200 CEST,ContainerID:cri-o://dc798098f24caa42cbb1d08eb507f921f43a5da74d32377f560209bbbd7e2c38,}
        sidecar_test.go:140: Either the primary or sidecar containers did not terminate
FAIL
-------
```

events :

```
33s         Warning   Failed      pod/sidecar-test-task-run-0-pod-993fa8   Error: container create failed: container_linux.go:336: starting container process caused "exec: \"sh\": executable file not found in $PATH"
```

# Steps to Reproduce the Problem

1. 
```
% go test -failfast -v -count=1 -tags=e2e -ldflags '-X github.com/tektoncd/pipeline/test.missingKoFatal=false' ./test -timeout=20m --kubeconfig $KUBECONFIG -run TestSidecarTaskSuppor
```

# Additional Info


```
% oc version
Client Version: version.Info{Major:"4", Minor:"1+", GitVersion:"v4.1.13-201908210601+15d7c4b-dirty", GitCommit:"15d7c4b", GitTreeState:"dirty", BuildDate:"2019-08-21T13:24:47Z", GoVersion:"go1.11.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.4+df9cebc", GitCommit:"df9cebc", GitTreeState:"clean", BuildDate:"2019-08-06T18:31:45Z", GoVersion:"go1.11.6", Compiler:"gc", Platform:"linux/amd64"}
```

--- a/test/sidecar_test.go
+++ b/test/sidecar_test.go
@@ -64,12 +64,12 @@ func TestSidecarTaskSupport(t *testing.T) {
 				tb.TaskSpec(
 					tb.Step(
 						primaryContainerName,
-						"busybox:1.31.0-musl",
+						"ubuntu",
 						tb.StepCommand(test.stepCommand...),
 					),
 					tb.Sidecar(
 						sidecarContainerName,
-						"busybox:1.31.0-musl",
+						"ubuntu",
 						tb.Command(test.sidecarCommand...),
 					),
 				),

